src/main/java/com/redhat/cloud/notifications/auth/RHIdentityAuthMechanism.java

```diff
@@ -49,7 +49,7 @@
                     good = true;
                 }
             } else if (path.startsWith("/openapi.json") || path.startsWith("/internal")
-                    || path.startsWith("/admin") || path.startsWith("/health") || path.startsWith("/q/health")) {
+                    || path.startsWith("/admin") || path.startsWith("/health")) {
```

Comment:
Can you add /metrics in here as well please?

Reply:
Sure, it's done.